src/lib.rs

```diff
@@ -485,6 +491,7 @@ pub use naive::{IsoWeek, NaiveDate, NaiveDateTime, NaiveTime, NaiveWeek};
 
 pub mod offset;
 #[cfg(feature = "clock")]
+#[cfg_attr(docsrs, doc(cfg(feature = "clock")))]
```

Comment:
It looks like this doesn't work with Re-exports

Reply:
or perhaps `no_inline` is causing the issue, as the requirement for `clock` feature doesn't show up in the prelude either

Reply:
I've tried toggling no_inline but doesn't seem to help, so I'm assuming it's a re-export thing

Reply:
I'm going to resolve this as it seems it's a rustdoc bug